src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/integration/resource/InboundEventsIntegrationTest.kt

```diff
@@ -166,13 +166,15 @@ class InboundEventsIntegrationTest : SqsIntegrationTestBase() {
 
     val bookingId = webTestClient.createBooking(courtBookingRequest, TEST_USERNAME)
 
+    prisonSearchApi().stubGetPrisoner(prisonNumber = "YD1234", prisonCode = "TRN", lastPrisonCode = WERRINGTON)
```

Comment:
On a transfer this is what the prison code actually looks like (TRN) if they have not yet be received into the prison they are being transferred to.